apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/Log4NetTest.cs

```diff
@@ -98,7 +99,7 @@ private class NoDelayScheduler : IScheduler
             Hierarchy hierarchy = null;
             if (requiresLog4Net)
             {
-                hierarchy = (Hierarchy)LogManager.GetRepository();
+                hierarchy = (Hierarchy)LogManager.GetRepository(Assembly.GetEntryAssembly());
```

Comment:
`Assembly.GetEntryAssembly()` returns null on netcore, causing tests to fail. Please use `typeof(Log4NetTest).GetTypeInfo().Assembly`.